src/components/Header/Header.tsx

```diff
@@ -60,6 +60,7 @@ export const Header = () => {
 
   return (
     <>
+      <script id="cookieyes" type="text/javascript" src="https://cdn-cookieyes.com/client_data/5f6661d434c5243eba8423d0/script.js" async />
```

Comment:
@sharlotta93 In NextJS, it's not allowed to import scripts like this. CookieEyes has a specific guide for NextJS 12, see [here](https://www.cookieyes.com/documentation/install-cookie-banner-on-next-js-12-and-below-versions/). Can you do those changes?

Reply:
I was following nextJS docs https://nextjs.org/docs/messages/no-sync-scripts (hence the async) but given we can't use free tier I will use another provider